CalibPPS/TimingCalibration/test/DiamondSampicCalibrationHarvester_cfg.py

```diff
@@ -27,7 +27,7 @@
     toPut = cms.VPSet(
         cms.PSet(
             record = cms.string('PPSTimingCalibrationRcd'),
-            tag = cms.string('DiamondSampicCalibration'),
```

Comment:
I think you might want to change the record name here too.

Reply:
I have a few problems with that file, like the 
https://github.com/cms-sw/cmssw/blob/fc6a963a563c3f1079b918c81868a24dca9b90b3/CalibPPS/TimingCalibration/test/DiamondSampicCalibrationHarvester_cfg.py#L22

or
https://github.com/cms-sw/cmssw/blob/fc6a963a563c3f1079b918c81868a24dca9b90b3/CalibPPS/TimingCalibration/test/DiamondSampicCalibrationHarvester_cfg.py#L48

or
https://github.com/cms-sw/cmssw/blob/fc6a963a563c3f1079b918c81868a24dca9b90b3/CalibPPS/TimingCalibration/test/DiamondSampicCalibrationHarvester_cfg.py#L38

so I think I'll just suggest PPS to delete that file as it is (in a follow-up PR, which is less urgent)

Reply:
Mmh, ok, but why changing it here?

Reply:
I fixed the record name